.nvmrc

```
@@ -0,0 +1 @@
+22
```

```suggestion
22.12
```

To match what we did for `electron/get`.